tools/pythonpkg/tests/fast/test_expression.py

```diff
@@ -0,0 +1,572 @@
+import duckdb
+import pytest
+from duckdb.typing import INTEGER, VARCHAR, TIMESTAMP
+from duckdb import Expression, ConstantExpression, ColumnExpression, StarExpression, FunctionExpression, CaseExpression
+from pyduckdb.value.constant import Value, IntegerValue
+import datetime
+
+
+@pytest.fixture(scope='function')
+def filter_rel():
+    con = duckdb.connect()
+    rel = con.sql(
+        """
+        select * from (VALUES
+            (1, 'a'),
+            (2, 'b'),
+            (1, 'b'),
+            (3, 'c'),
+            (4, 'a')
+        ) tbl(a, b)
+    """
+    )
+    yield rel
+
+
+class TestExpression(object):
+    def test_constant_expression(self):
+        con = duckdb.connect()
+
+        val = Value(5, INTEGER)
+
+        rel = con.sql(
+            """
+            select
+                1 as a,
+                2 as b,
+                3 as c
+        """
+        )
+
+        constant = ConstantExpression(val)
+
+        rel = rel.select(constant)
+        res = rel.fetchall()
+        assert res == [(5,)]
+
+    def test_column_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+            select
+                1 as a,
+                2 as b,
+                3 as c
+        """
+        )
+        column = ColumnExpression('a')
+        rel2 = rel.select(column)
+        res = rel2.fetchall()
+        assert res == [(1,)]
+
+        column = ColumnExpression('d')
```

Comment:
Add a test where we explain the logical plan to verify the presence of the expressions we created

Also check behavior on a table/view/temporary table, replacement scan etc for extra coverage